src/main/kotlin/thatline/localup/auth/service/EmailService.kt

```
@@ -0,0 +1,127 @@
+    private fun sendEmail(to: String, subject: String, htmlContent: String) {
+        try {
+            val mimeMessage = mailSender.createMimeMessage()
+            val helper = MimeMessageHelper(mimeMessage, true, "UTF-8")
+
+            helper.setFrom(FROM_EMAIL)
+            helper.setTo(to)
+            helper.setSubject(subject)
+            helper.setText(htmlContent, true) // HTML 형식으로 설정
+
+            mailSender.send(mimeMessage)
+        } catch (exception: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)